packages/framework/aqueduct/src/data-objects/treeDataObject.ts

```diff
@@ -50,6 +50,23 @@ export abstract class TreeDataObject<TTreeView> extends PureDataObject {
 	 */
 	protected abstract generateView(tree: ITree): TTreeView;
 
+	/**
+	 * Implementation of SharedTree which is used to generate the view.
+	 * @remarks Created once during initialization.
+	 */
+	#sharedTree: ITree | undefined;
+
+	/**
+	 * Gets the underlying tree.
+	 * @remarks Used to give access to the underlying tree and its schema.
```

Comment:
Nit: I don't think this offers much additional context over the summary. We can probably remove this IMO.